### DISCUSSION TITLE
RGBW2 Dim Values

### DISCUSSION THREAD

**Discussion by sroettgermann** [mongoose-os-apps/shelly-homekit#1485] [Ideas]
Just curious if it is possible to adjust the dimmer values? For my LED setup (bulbs) the brightness only changes between 0% (off) and 15% (full brightness). I’d like to set up some kind of a correction value.

**Comment by mblackshaw** [comment]
+1

None of my bulbs actually use the dimmer range. Being able to define the max brightness would be super useful!
